content/references/jwt-la-gi/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: JWT là gì
+description: JWT laf moot
```

Comment:
@nguyenvanduocit I suggest clarifying the description or leave it blank

> JWT là một phương thức xác thực gọn nhẹ

Reply:
Absolutely